.github/workflows/pr.yml

```diff
@@ -162,4 +164,11 @@ jobs:
       - cccl-infra
       - verify-devcontainers
     steps:
-      - run: echo "CI success"
```

Comment:
Comment: Do we want to keep a "CI success" echo anywhere? Nbd either way, just noting that it's gone now.

Reply:
Nah, I think we just had that because we couldn't have empty `steps`.